index.html

```diff
@@ -530,27 +590,65 @@ <h3>Script Event</h3>
+                <p class="note">If a <code>&lt;p&gt;</code> element omits the <code>xml:lang</code> attribute then its computed language
+                is derived by inheritance from its parent element, and so forth up to the root <code>&lt;tt&gt;</code> element,
+                which is required to set the <a>Primary Language</a> via its <code>xml:lang</code> attribute.
+                Care should be taken if changing the <a>Primary Language</a> of a <a>Document Instance</a> in case
+                doing so affects descendant elements unexpectedly.
+                Authors can mitigate this risk by explicitly setting <code>xml:lang</code> on all <code>&lt;p&gt;</code> elements.</p>
               </li>
           <pre class="example">
 &lt;div xml:id=&quot;event_3&quot;
      begin=&quot;9663f&quot; end=&quot;9682f&quot; 
      style=&quot;style_a&quot;
      ttm:agent=&quot;character_3&quot;&gt;
-  &lt;p xml:lang=&quot;pt-BR&quot; &gt;Voc&#xea; vai ter.&lt;/p&gt;
-  &lt;p xml:lang=&quot;fr&quot; ttm:role=&quot;x-context&quot;&gt;Bah, il arrive.&lt;/p&gt;
+  &lt;p xml:lang=&quot;pt-BR&quot; daptm:tls=&quot;original&quot; &gt;Voc&#xea; vai ter.&lt;/p&gt;
+  &lt;p xml:lang=&quot;fr&quot; daptm:tls=&quot;translation&quot; &gt;Bah, il arrive.&lt;/p&gt;
 &lt;/div&gt;
           </pre>
         </section>
 
+        <section>
+          <h4>Text Language Source</h4>
+          <p>The <dfn>Text Language Source</dfn> property is an annotation indicating whether a <a>Text</a> object is
+            in the same language as the relevant part of the <a>Related Media Object</a>'s
+            language (original), or if it is a representation in another language (translation):</p>
+          <ul>
+            <li><dfn>Original</dfn> - the <a>Text</a> is in the same language as the dialogue that it represents in the original programme audio,
+              or it represents non-dialogue sound and is in the <a>Primary Language</a> of the <a>Document Instance</a>.</li>
+            <li><dfn>Translation</dfn> - the <a>Text</a> is a representation of the Original language text in a different language.</li>
+          </ul>
+          <aside class="note">The language of a <a>Text</a> object whose <a>Text Language Source</a> is <a>Original</a>
+            can be different to the document's <a>Primary Language</a>.
+            For example a transcript of a short piece of dialogue spoken in a different language to the majority of the dialogue would be annotated
+            with both a different <code>xml:lang</code> value and as being <a>Original</a>.</aside>
+          <aside class="note"><a>Original</a> text is known as "Main",
+            and <a>Translation</a> text as "Contextual", in some workflows.</aside>
+          <p>The <a>Text Language Source</a> property is represented as a <code>daptm:tls</code> attribute with the following constraints:</p>
+          <p class="ednote">Should we use an abbreviated attribute name?</p>
```

Comment:
maybe simply `dapt:languageSource`?

Reply:
OK, that works - my only query is whether it makes sense to make frequently used attribute names shorter, but I think semantic clarity is more important.

Reply:
`langSrc`?

Reply:
Done in 3cfe8db.